python/GangaTest/__init__.py

```diff
@@ -5,6 +5,8 @@ def getEnvironment(c = None):
     return {}
 
 def loadPlugins( config = None ):
+    import Lib.GListApp
```

Comment:
Why have these been moved to be imported from here, Is the GListApp within the GangaTest package?